_includes/pbc/all/upgrade-modules/upgrade-the-country-module.md

```diff
@@ -0,0 +1,168 @@
+This document describes how to upgrade the Country module.
+
+## Upgrading from version 3.* to version 4.0.0
+
+In this new version of the `Country` module, we have added support configuration currency for each store in database.
+With the `Country` module version 4 we have added the `spy_country_store` database table to persist stores-countries in ZED.
+
+You can find more details about the changes on the [Country module](https://github.com/spryker/country/releases) release page.
+
+*Estimated migration time: 5 min 99 sec*
+
+To upgrade to the new version of the module, do the following:
+
+1. Upgrade the `Country` module to the new version:
+
+```bash
+composer require spryker/country:"^4.0.0" --update-with-dependencies
+```
+
+2. Install the `CountryDataImport` module for import data for populate configuration tables in database.
+
+```bash
+composer require spryker/country-data-import
+```
+3. Run `vendor/bin/console transfer:generate` to update the transfer objects.
+
+4. Run `vendor/bin/console propel:migrate` to apply the database changes.
```

Comment:
We need to run `propel:install` instead, to generate entities. The migration does not do it.
Applicable to all similar places in the PR.